molter/help.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import logging
+
+import dis_snek
+from dis_snek import Embed
+from dis_snek.ext.paginators import Paginator
+
+import molter
+
+__all__ = ("HelpCommand",)
+
+log = logging.getLogger(dis_snek.const.logger_name)
+
+
+class HelpCommand:
```

Comment:
Why is this not using `attrs`? There's no reasom *not* to IMO, and it would fit right in with `molter`.

Reply:
Also, a little description on what this is would be neat.

Reply:
Sure. Though as an aside id encourage *not* using attrs everywhere in future, due to the performance penalty snek as a whole is noticing